automated install/basic-install.sh

```diff
@@ -183,27 +183,23 @@ os_check() {
     # and determines whether or not the script is running on one of those systems
     local remote_os_domain valid_os valid_version detected_os_pretty detected_os detected_version display_warning
```

Comment:
This should be cleaned up as some variables don't exist anymore

Reply:
Thanks, good catch!